src/hardware/input/mouse_config.cpp

```diff
@@ -362,6 +371,19 @@ static void config_init(Section_prop &secprop)
 	        "  3button+msm:  Automatic choice between '3button' and 'msm'.\n"
 	        "  wheel+msm:    Automatic choice between 'wheel' and 'msm' (default).\n"
 	        "Note: Enable COM port mice in the [serial] section.");
+
+	// VMM interfaces
+
+#ifdef EXPERIMENTAL_VIRTUALBOX_MOUSE
+
+	prop_bool = secprop.Add_bool("vmware_mouse", only_at_start, true);
+	prop_bool->Set_help("VMware mouse interface (enabled by default).\n"
+	                    "Note: it requires PS/2 mouse to be enabled.\n");
```

Comment:
"Note: This requires..."

(Start sentence with uppercase, and "this" reads better).